nowcasting_dataset/data_sources/nwp/nwp_data_source.py

```diff
@@ -66,13 +74,13 @@ def open(self) -> None:
         """
         Open NWP data
 
-        We don't want to open_sat_data in __init__.
+        We don't want to open_nwp() in __init__.
         If we did that, then we couldn't copy NWPDataSource
         instances into separate processes.  Instead,
         call open() _after_ creating separate processes.
         """
         data = self._open_data()
-        self._data = data["UKV"].sel(variable=list(self.channels))
```

Comment:
how did that sneak in? - if it was me - thanks for fixing that

Reply:
don't worry!  I just changed `open_nwp()` so it now returns a DataArray, not a Dataset!